repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -139,9 +143,12 @@ def scan_events(path):
     """ Scan JSON file containing PES events """
     try:
         events = parse_file(path)
-    except (ValueError, KeyError) as error:
-        raise StopActorExecutionError(
-            message='Found invalid or outdated PES data.',
-            details={'details': error, 'hint': 'Consult docs to retrieve updated data.'})
+    except (ValueError, KeyError) as value_error:
+        title = 'Missing PES data ({})'.format(path)
+        summary = 'Read documentation at: https://access.redhat.com/articles/3664871 for more information ' \
+            'about how to retrieve the files'
+        error = str(value_error)
```

Comment:
Since we are using this only once, I would vote for dropping it and using "str(value_error)" directly in place